### ISSUE TITLE
Ban Reason is empty ('*') if IP is in Skynet-BlockedRanges?

### ISSUE THREAD

**Issue by lhchang** [Adamm00/IPSet_ASUS#90]
I've been running Skynet out of the box with no manual banlists or anything of the sorts. Noticed I had a outbound connection blocked but there was no reason given. I did see the IP range listed in Skynet-BlockedRanges from the et_block set. I was wondering if this was intentional that there does not exist a ban reason for this entry despite it being in an ipset, or if I am just missing something?

Thank you and appreciate any clarifications.

**Comment by Adamm00** [comment]
Skynet uses purely bash, so that limits us in some minor ways. One of those being easy CIDR calculations for ban reason lookups. I could improve this but the code would be huge and not worth the minor benefit it adds.

TL;DR Skynet isnt smart enough to search for IP's in **_some_** edge cases where large CIDR's are used.
